src/wallet/wallet.cpp

```diff
@@ -5214,11 +5221,13 @@ void CWallet::AvailableCoins(vector<COutput>& vCoins,
                             i,
                             hasDestination ? std::optional(address) : std::nullopt,
                             nDepth,
-                            isSpendable,
+                            consideredSpendable,
```

Comment:
For the current PR, this will always be `true`, because we `continue` when it is false and never reach here. That slightly changes how this API works: previously it would report whether the output would have been selected if spendability were a question; now it doesn't make that distinction when the filter is set to select unspendable outputs.

I feel like we need both arguments here: the old `fOnlySpendable` boolean, and the new filter that is used to define what spendability _is_.